### PR TITLE
Adds toycars and their remote control.

### PR DESCRIPTION
**Pull request by Aranclanos**
cars

### CODE REVIEW

**Comment by Chocobro** [comment]
*confetti*
:+1:

**Comment by Cheridan** [comment]
#7000

**Comment by Jalleo** [comment]
@Iamgoofball welp

File: remotecar.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+/obj/item/toycarremote/self_attack(user)
+	user.client.eye = car
+
+/obj/item/toycarremote/process()
+	if(not stunned or something)
+		return
+	fixhiseye
+
+//i swear i didn't do this in 2 minutes
```
Comment (Anonus) [inline remotecar.dm]: Three?

**Comment by GrayRachnid** [comment]
#6999

File: remotecar.dm [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/obj/item/toycarremote
+	var/obj/item/toycar/car
+
+/obj/item/toycarremote/self_attack(user)
+	user.client.eye = car
+
+/obj/item/toycarremote/process()
+	if(not stunned or something)
+		return
+	fixhiseye
```
Comment (GrayRachnid) [inline remotecar.dm]: If only this works..

**Comment by ike709** [comment]
aran this is why people don't like you

**Comment by ike709** [comment]
>#7000 needs to be book stocking mech

somebody didn't listen to #6000's comments apparently

**Comment by paprka** [comment]
Can we ban aran from contributing

**Comment by Iamgoofball** [comment]
You better fucking finish this one or I'm making #8000 be cattebeest serious edition

**Comment by ike709** [comment]
please finish #5000, #6000, and this

**Comment by hornygranny** [comment]
you couldn't even put in 15 minutes of effort to make this simple meme actually work

**Pull request closed without merging**